modules/apps/transfer/types/keys.go

```diff
@@ -29,6 +29,9 @@ const (
 
 	// DenomPrefix is the prefix used for internal SDK coin representation.
 	DenomPrefix = "ibc"
+
+	KeyTotalEscrowPrefix = "totalEscrow"
+	KeyDenomsPrefix      = "denoms"
```

Comment:
It might make sense for this to be a single key? Is there a reason to have 2?

Reply:
The reason for 2 keys is to follow the pattern that we use for other paths, e.g. `channelEnds/ports/{identifier}/channels/{identifier}`.

Reply:
I think I'd prefer it to be one key, I don't find the pattern used for channel/port id path's to be equivalent. In referenced case, `channelEnds` is a key for the 04-channel keeper. Here the `totalEscrow` and `denoms` are referencing the same key, is there a scenario when we use `totalEscrow` as a prefix for a different key? Otherwise I'd prefer it to just be `totalEscrow/{denom}`

Reply:
I see your point, thanks! And what about if the keys are swapped, so `denoms/total_escrow/{denom}`? In case in the future other information for denoms is stored we can still iterate over the `denoms/total_escrow` to filter any other information. Would that be useful?